src/java.base/share/classes/javax/crypto/SealedObject.java

```diff
@@ -442,13 +440,13 @@ private ObjectInputStream getExtObjectInputStream(Cipher c)
     }
 
     static {
-        SharedSecrets.setJavaxCryptoSealedObjectAccess((obj,c) -> obj.getExtObjectInputStream(c));
+        SharedSecrets.setJavaxCryptoSealedObjectAccess(SealedObject::getExtObjectInputStream);
     }
 }
 
 final class extObjectInputStream extends ObjectInputStream {
     extObjectInputStream(InputStream in)
-        throws IOException, StreamCorruptedException {
+        throws IOException {
```

Comment:
This is another "I probably wouldn't do that..."  

It's nice to know what kind of IOExceptions you might get, so leaving StreamCorruptedException is ok.

Reply:
Keeping the original. Still, why would IJ suggest removing this unless it had already figured out the exception could never happen?